### PR TITLE
feature/createplaylistwithoutnameexception

### PR DESCRIPTION
**Pull request by hemasreedarisiguntla**
1. Refactored the uri for creating new playlist and name is implemented as Request Param.
2. implemented feature to provide appropriate message when name is not supplied.

### CODE REVIEW

File: src/main/java/com/musicshare/playlistapi/controller/PlayListController.java [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-
 @RestController
 @RequestMapping("/api/v1/playlist")
 public class PlayListController {
 
     @Autowired
     PlayListService playListService;
 
-    @PostMapping("/{name}")
+    @PostMapping()
```
Comment (shashi2k6) [inline src/main/java/com/musicshare/playlistapi/controller/PlayListController.java]: Please remove the braces

File: src/main/java/com/musicshare/playlistapi/controller/PlayListController.java [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
     @ResponseStatus(HttpStatus.CREATED)
```
Comment (shashi2k6) [inline src/main/java/com/musicshare/playlistapi/controller/PlayListController.java]: Return the success response in successfully creation of playlist.

**Review by shashi2k6 — CHANGES_REQUESTED** [review]
Please implement the review comments.

File: src/main/java/com/musicshare/playlistapi/controller/PlayListController.java (hunk 1 above)
Comment (hemasreedarisiguntla) [reply src/main/java/com/musicshare/playlistapi/controller/PlayListController.java]: Removed the braces

File: src/main/java/com/musicshare/playlistapi/controller/PlayListController.java (hunk 2 above)
Comment (hemasreedarisiguntla) [reply src/main/java/com/musicshare/playlistapi/controller/PlayListController.java]: It seems that we cannot implement reason for success response.

**Pull request merged**